sw/service-worker.ts

```diff
@@ -0,0 +1,84 @@
+import { clientsClaim, skipWaiting } from 'workbox-core';
+import { ExpirationPlugin } from 'workbox-expiration';
+import { cleanupOutdatedCaches, precacheAndRoute } from 'workbox-precaching';
+import { registerRoute, setDefaultHandler } from 'workbox-routing';
+import { CacheFirst, NetworkFirst, StaleWhileRevalidate } from 'workbox-strategies';
+import { CacheChildren } from './CacheChildrenStrategy';
+
+
+// Setting the default expiration options for the caches.
+const DEFAULT_EXPIRATION_OPTIONS = {
+  maxEntries: 128,
+  maxAgeSeconds: 60 * 60 * 24,
+  purgeOnQuotaError: true,
+};
```

Comment:
since we're using stale-while-revalidate, a request for a github issue is always up to date in the cache. Do we need to expire it at all?

Reply:
but it's not a sticky cache since cache is *always* overridden on new requests. the worst case scenario is two refreshes to see updates.